miasm2/jitter/jitload.py

```diff
@@ -134,13 +137,19 @@ class CallbackHandlerBitflag(CallbackHandler):
 
     "Handle a list of callback with conditions on bitflag"
 
+    # Overrides CallbackHandler's implem, but do not serve for optimization
+    def has_callbacks(self, bitflag):
```

Comment:
This function can be rewritten as (more readable, in my opinion):
```Python
    return any(b & bitflag != 0 for b in self.callbacks)
```
This function would return either `True` or `False` instead of either `True` or `None`, by the way.